docs/source/adb.rst

```diff
@@ -0,0 +1,136 @@
+.. _adb:
+
+####################
+Android Debug Bridge
+####################
+
+.. important:: Android Debug Bridge (adb)is required ONLY for those users pairing an Android mobile device with Cozmo. iOS users require a separate utility called *usbmuxd*; instructions for installing this, if necessary, are given on the install pages for each operating system.
```

Comment:
I think the entire second sentence ("iOS users require") should be removed, since this page is specifically for Android users, and since iOS users only actually directly install usbmuxd on Linux.